project.clj

```diff
@@ -62,7 +62,7 @@
                                   [puppetlabs/trapperkeeper ~tk-version :classifier "test"]
                                   [org.clojure/tools.namespace "0.2.4"]
                                   [org.clojure/java.jmx "0.2.0"]
-                                  [spyscope "0.1.4"]
+                                  [spyscope "0.1.5" :exclusions [clj-time]]
```

Comment:
That's fine, but FYI I've been hitting issues related to clj-time in other projects as well, and I think it's probably also an OK idea to just explicitly specify a version of clj-time for the project.  (I think one of my later J9 PRs does this.)